typeform/typeform.py

```diff
@@ -0,0 +1,182 @@
+import time
+import copy
+import json
+import panoply
+import urllib2
+import dateparser
+
+
+FETCH_LIMIT = 500
+DESTINATION = 'typeform'
+DESTINATION_POSTFIX = '_{__table}'
+BASE_URL = 'https://api.typeform.com/v1'
+
+
+class Typeform(panoply.DataSource):
+
+    # constructor
+    def __init__(self, source, options):
+        super(Typeform, self).__init__(source, options)
+
+        if 'destination' not in source:
+            source['destination'] = DESTINATION
+
+        # append the destination postfix, which represent the form name
+        source['destination'] += DESTINATION_POSTFIX
+
+        # configure incremental pulling
+        self._incval = None
+        if (source.get('inckey')):
+            incval = source.get('incval')
+            self._incval = (incval if incval else
+                source.get('lastTimeSucceed') or source.get('lastRuntime'))
```

Comment:
I think this is a bit too much to grasp, would this be the same thing?
`self._incval = incval or source.get('lastTimeSucceed') or source.get('lastRuntime')`